rq_dashboard/__init__.py

```diff
@@ -1,19 +1,2 @@
-# Modules/blueprints
-from .dashboard import dashboard
-
-
-class RQDashboard(object):
-    def __init__(self, app=None, url_prefix='/rq', auth_handler=None):
-        self.url_prefix = url_prefix
-        if app is not None:
-            self.app = app
-            self.init_app(app)
-        else:
-            self.app = None
-        self.auth_handler = auth_handler
-        self.redis_conn = None
-
-    def init_app(self, app):
-        """Initializes the RQ-Dashboard for the specified application."""
-        app.register_blueprint(dashboard, url_prefix=self.url_prefix)
-        app.extensions['rq-dashboard'] = self
+import default_settings  # NOQA
+import blueprint  # NOQA
```

Comment:
Yes I agree that from e.g. scripts/rq_dashboard.py that would create a nicer API. But I felt it would be surprising and may even lead to circular import problems. Perhaps either the `blueprint.py` module or the `blueprint` variable should be named differently? Perhaps the module should be called `web.py`, especially if we also factor out the RQ status to (say) `rqstatus.py`.

Then the shortcut in `__init__.py` as you suggest would work nicely.

What do you think?